pkg/api/index.go

```diff
@@ -513,6 +513,19 @@ func (hs *HTTPServer) buildAlertNavLinks(c *models.ReqContext, uaVisibleForOrg b
 		})
 	}
 
+	if hs.Features.IsEnabled(featuremgmt.FlagNewNavigation) {
+		if uaVisibleForOrg {
+			alertChildNavs = append(alertChildNavs, &dtos.NavLink{
+				Text: "Divider", Divider: true, Id: "divider", HideFromTabs: true,
+			})
+
+			alertChildNavs = append(alertChildNavs, &dtos.NavLink{
+				Text: "Alert rule", SubTitle: "Create an alert rule", Id: "alert",
+				Icon: "plus", Url: hs.Cfg.AppSubURL + "/alerting/new", HideFromTabs: true, ShowIconInNavbar: true,
+			})
+		}
+	}
```

Comment:
Think you also need to check if the user has permission (including fine-grain access control)